lib/rails/controller/testing.rb

```diff
@@ -10,16 +11,19 @@ def self.install
         ActiveSupport.on_load(:action_controller_test_case) do
           include Rails::Controller::Testing::TestProcess
           include Rails::Controller::Testing::TemplateAssertions
+          ActionView::TemplateRenderer.prepend(InstrumentDetermineTemplate)
```

Comment:
We should include this inside an `on_load(:action_view)` otherwise we could prepend this same module three times to the same class when running tests.